src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,99 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (newMovie: Movie) => void
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+
+  const [titleValue, setTitleValue] = useState('');
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
```
The names can be simplified. No need to repeat "value" everywhere